### DISCUSSION TITLE
How to get started with contribution......

### DISCUSSION THREAD

**Discussion by raja2825** [EddieHubCommunity/LinkFree#9383] [Q&A] [answered]
hey,i would like to contibute here......but i dont know how to get started.....can anyone help me with this

**Comment by ikrishg** [comment, marked as answer]
Hi! I suggest you to:

- Read the [contributing guide](https://github.com/EddieHubCommunity/BioDrop/blob/main/CONTRIBUTING.md)
- Look at our [open issues](https://github.com/EddieHubCommunity/BioDrop/issues)
- Join us at [EddieHub Discord](https://discord.gg/Bt9EQDSuQX) and look for upcoming events where a lot of contribution opportunities are dropped

I would be glad to help you if you need, any ping me @xkrishguptaa at the eddiehub server anytime in the #help channel! :D
